client/webserver/site/src/js/locales.ts

```diff
@@ -189,7 +191,9 @@ export const enUS: Locale = {
   [ID_MATCH_STATUS_REDEEM_PENDING]: 'Redeem PENDING',
   [ID_MATCH_STATUS_REDEMPTION_CONFIRMED]: 'Redemption Confirmed',
   [ID_MATCH_STATUS_COMPLETE]: 'Complete',
-  [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'redeemed by {{ makerAddr }}'
+  [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'redeemed by {{ makerAddr }}',
+  [ID_REFUND_IMMINENT]: 'Coming your way, just a little patience', // alt: Will happen in the next few blocks
+  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Will happen after {{ refundAfterTime }}'
```

Comment:
imo:
(Capitalization)
redeemed by -> Redeemed by
(More professional)
Coming your way, just a little patience -> Will happen in the next few blocks
(More easily understandable with the noun)
Will happen after -> Refund will happen after

Reply:
> Will happen after -> Refund will happen after

I started with this one actually, but repeated Refund (in title, and actual body) kinda scared me off.

Will apply, if nobody has other suggestions.

Reply:
Updated: https://github.com/decred/dcrdex/pull/2042/commits/3dc2b757f598339ce4cb241afc66fef6b28f387b